recipe/bld.bat

```diff
@@ -27,9 +27,9 @@ if errorlevel 1 exit 1
     variant=release ^
     threading=multi ^
     link=static,shared ^
-    -j%CPU_COUNT% ^
-    -s ZLIB_INCLUDE="%LIBRARY_INC%" ^
-    -s ZLIB_LIBPATH="%LIBRARY_LIB%"
+    -j%CPU_COUNT%
+::  -s ZLIB_INCLUDE="%LIBRARY_INC%" ^
+::  -s ZLIB_LIBRARY_PATH="%LIBRARY_LIB%"
```

Comment:
* So is this done automatically by Boost now?
* Please add a comment on why this is not needed anymore :-)